components/21-atoms/table/table.bindings.js

```diff
@@ -0,0 +1,21 @@
+/** eslint-disable no-console */
+
+'use strict';
+
+(function () {
+  var tablesNodeList = document.querySelectorAll('.responsive-table');
+
+  // Optimise all tables with a wrapper div.responsive-table
+  for (var i = 0; i < tablesNodeList.length; i++) {
+    var table = tablesNodeList[i];
+
+    // Adds accessibility support.
+    new ResponsiveTable(table, { // eslint-disable-line no-undef
```

Comment:
```suggestion
    new ResponsiveTable(table, {
```

Reply:
the eslint-disable isn't necessary anymore since you declared it as a global variable at the top of this file